index.php

```diff
@@ -527,11 +527,19 @@ class="dt">&#148;</span>
 				$rating = explode(" ", $photos[$p]);
 				$hits = $rating[0];
 				$hits++;
+				//secondary version:
+				$rate = 0;
+				$raters = substr(strrchr($rating[1], '/'), 1);
+				if ($raters != "0")
 				@eval("@\$rate =".$rating[1].";");
-				if (($rate < 1) || ($rate > 5))
-					$rating[1] = "0/0";
-				if (isset($rating[2]))
-					$curLastVisit = $rating;
+				$rate = round($rate, 2);
+				
+				//primary format:
+				//@eval("@\$rate =".$rating[1].";");
+				// if (($rate < 1) || ($rate > 5))
+				// 	$rating[1] = "0/0";
+				// if (isset($rating[2]))
+				// 	$curLastVisit = $rating;

```

Comment:
Why are we commenting these lines out? 🤔

Reply:
I wanted to avoid losing the original version of this piece. However, I realized that I could review the previous versions, so I removed the comments.

Reply:
It seems like you removed those lines instead of un-commenting them (to become like the original version).

Reply:
That's right. I'm sorry for the mistake.